github/src/main/proto/spine_examples/pingh/github/github.proto

```diff
@@ -58,6 +59,25 @@ message User {
     // The unique username that identifies the user.
     Username username = 1 [(required) = true];
 
-    // The link to the user's avatar.
+    // The URL to the user's avatar.
     spine.net.Url avatar_url = 2 [(required) = true];
 }
+
+// The mention of a user on the GitHub.
```

Comment:
Usually, names are not used with "the". There are exceptions, such as "The United States of America". But I don't think GitHub is.